packages/webapp/src/routes/land/legal-agreement-lpa-remove.js

```diff
@@ -0,0 +1,59 @@
+import constants from '../../utils/constants.js'
+import { buildFullName, processRegistrationTask } from '../../utils/helpers.js'
+const handlers = {
+  get: async (request, h) => {
+    processRegistrationTask(request, {
+      taskTitle: 'Legal information',
+      title: 'Add legal agreement details'
+    }, {
+      inProgressUrl: constants.routes.LEGAL_AGREEMENT_LPA_REMOVE
+    })
+
+    const { id } = request.query
```

Comment:
GET handler joi validation for query string?

Reply:
new ticket